Ginger/GingerCoreNET/ActionsLib/General/ActSetVariableValue.cs

```diff
@@ -225,7 +234,16 @@ public override void Execute()
             }
             else if (SetVariableValueOption == VariableBase.eSetValueOptions.AutoGenerateValue)
             {
-                ((VariableBase)Var).GenerateAutoValue();
+                isAutoGenerateValuesucceed = ((VariableBase)Var).GenerateAutoValue(ref errorMsg);
```

Comment:
Remove all "Status" value set, the status is been calculated by GingerExecutionEngin